apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts

```diff
@@ -824,60 +824,55 @@ export const functions: NavMenuConstant = {
       name: 'Overview',
       url: '/guides/functions',
     },
+    ,
     {
       name: 'Quickstart',
-      url: '/guides/functions/quickstart',
-    },
-    {
-      name: 'Features',
       url: undefined,
       items: [
-        { name: 'TypeScript Support', url: '/guides/functions/typescript-support' },
-        { name: 'Debugging Edge Functions', url: '/guides/functions/debugging' },
-        { name: 'Managing packages using Import Maps', url: '/guides/functions/import-maps' },
+        {
+          name: 'Getting started',
+          url: '/guides/functions/quickstart',
+        },
+        {
+          name: 'Deploy to Production',
+          url: '/guides/functions/deploy',
+        },
+        { name: 'Setting up your editor', url: '/guides/functions/local-development' },
       ],
     },
     {
       name: 'Guides',
       url: undefined,
       items: [
-        { name: 'Developing Functions locally', url: '/guides/functions/local-development' },
-        { name: 'Deploying with GitHub', url: '/guides/functions/cicd-workflow' },
-        { name: 'Managing Secrets and Environment Variables', url: '/guides/functions/secrets' },
-        { name: 'Integrating With Supabase Auth', url: '/guides/functions/auth' },
+        { name: 'Managing dependencies', url: '/guides/functions/import-maps' },
+        { name: 'Managing Environment Variables', url: '/guides/functions/secrets' },
+        { name: 'Integrating with Supabase Auth', url: '/guides/functions/auth' },
         {
-          name: 'Integrating with Supabase Storage',
-          url: '/guides/functions/storage-caching',
+          name: 'Integrating with Postgres',
+          url: '/guides/functions/connect-to-postgres',
         },
-        { name: 'CORS support for Invoking from the browser', url: '/guides/functions/cors' },
-        { name: 'Scheduling Functions', url: '/guides/functions/schedule-functions' },
         {
-          name: 'Connecting directly to Postgres',
-          url: '/guides/functions/connect-to-postgres',
+          name: 'Integrating with Supabase Storage',
+          url: '/guides/functions/storage-caching',
         },
-        { name: 'Testing your Edge Functions', url: '/guides/functions/unit-test' },
-        { name: 'Regional Invocation', url: '/guides/functions/regional-invocation' },
-        { name: 'Troubleshooting', url: '/guides/functions/troubleshooting' },
+        { name: 'Regional invocations', url: '/guides/functions/regional-invocation' },
```

Comment:
nit: Page title for it is still `Regional Invocation`. Let's change that as well?

Reply:
done 👍